packages/appsync-modelgen-plugin/src/visitors/appsync-javascript-visitor.ts

```diff
@@ -60,7 +64,7 @@ export class AppSyncModelJavascriptVisitor<
         .map(typeObj => this.generateModelDeclaration(typeObj, true))
         .join('\n\n');
 
-      return [imports, enumDeclarations, nonModelDeclarations, modelDeclarations].join('\n\n');
+      return [imports, enumDeclarations, nonModelDeclarations, modelMetaData, modelDeclarations].join('\n\n');
```

Comment:
Again, inserting a new field like modelMetaData in the middle of an array is not ideal for backward compatibility.